### PR TITLE
Create pi.hi.md

### PR DESCRIPTION
**Pull request by AdityaGarg00**

### CODE REVIEW

File: 1-getting-started/lessons/1-introduction-to-iot/translations/pi.hi.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    ![ग्रोव हैट फिट करना](../../../../images/pi-grove-hat-fitting.gif)
+
+1. तय करें कि आप अपने पाई को कैसे प्रोग्राम करना चाहते हैं, और नीचे दिए गए संबंधित अनुभाग पर जाएं:
+
+    * [सीधे अपने पाई पर काम करें](#सीधे-अपने-पाई-पर-काम-करें)
+    * [पाई को कोड करने के लिए रिमोट एक्सेस](#पाई-को-कोड-करने-के-लिए-रिमोट-एक्सेस)
+
+### सीधे अपने पाई पर काम करें
+
+यदि आप सीधे अपने पीआई पर काम करना चाहते हैं, तो आप रास्पबेरी पाई ओएस के डेस्कटॉप संस्करण का उपयोग कर सकते हैं और अपने सभी आवश्यक टूल्स इंस्टॉल कर सकते हैं।
```
Comment (Anurag-0-1-A) [inline 1-getting-started/lessons/1-introduction-to-iot/translations/pi.hi.md:30]: I think पीआई will be replaced by पाई

File: 1-getting-started/lessons/1-introduction-to-iot/translations/pi.hi.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+1. [वीएस कोड प्रलेखन](https://code.visualstudio.com?WT.mc_id=academic-17441-jabenn) का पालन करके अपने कंप्यूटर पर वीएस कोड स्थापित करें।
+
+1. आवश्यक घटकों को स्थापित करने के लिए [एसएसएच दस्तावेज़ीकरण का उपयोग कर वीएस कोड रिमोट डेवलपमेंट](https://code.visualstudio.com/docs/remote/ssh?WT.mc_id=academic-17441-jabenn) में दिए गए निर्देशों का पालन करें।
+
+1. समान निर्देशों का पालन करते हुए, VS कोड को पाइ से कनेक्ट करें
+
+1. कनेक्ट होने के बाद, [पाइलेंस एक्सटेंशन](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance&WT.mc_id=academic-17441-jabenn) इंस्टॉल करने के लिए [मैनेजिंग एक्सटेंशन](https://code.visualstudio.com/docs/remote/ssh#_managing-extensions?WT.mc_id=academic-17441-jabenn) निर्देशों का पालन करें।
+
+## हैलो वर्ल्डा
```
Comment (Anurag-0-1-A) [inline 1-getting-started/lessons/1-introduction-to-iot/translations/pi.hi.md:194]: Replace हैलो वर्ल्डा with हैलो वर्ल्ड in the heading and in line 200 also

**Review by Anurag-0-1-A — CHANGES_REQUESTED** [review]
Great Work !!

Just some minor changes are required.

File: 1-getting-started/lessons/1-introduction-to-iot/translations/pi.hi.md (hunk 1 above)
Comment (Anurag-0-1-A) [reply 1-getting-started/lessons/1-introduction-to-iot/translations/pi.hi.md:30]: Kindly check Line 30 again for resolving this error.

File: 1-getting-started/lessons/1-introduction-to-iot/translations/pi.hi.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+
+    पाई को अपडेट और रीबूट किया जाएगा। Pi के रीबूट होने पर `ssh` सत्र समाप्त हो जाएगा, इसलिए इसे लगभग 30 सेकंड के लिए छोड़ दें और फिर से कनेक्ट करें।
+
+1. फिर से जुड़े `ssh` सत्र से, ग्रोव हार्डवेयर के लिए सभी आवश्यक लाइब्रेरीज़ स्थापित करने के लिए निम्न आदेश चलाएँ:
+
+    ```sh
+    curl -sL https://github.com/Seeed-Studio/grove.py/raw/master/install.sh | sudo bash -s -
+    ```
+
+    पायथन की शक्तिशाली विशेषताओं में से एक [पिप पैकेज](https://pypi.org) स्थापित करने की क्षमता है - ये अन्य लोगों द्वारा लिखे गए कोड के पैकेज हैं और इंटरनेट पर प्रकाशित होते हैं। आप अपने कंप्यूटर पर एक कमांड के साथ एक पिप पैकेज स्थापित कर सकते हैं, फिर उस पैकेज को अपने कोड में उपयोग कर सकते हैं। यह ग्रोव इंस्टाल स्क्रिप्ट उन पिप पैकेजों को स्थापित करेगा जिनका उपयोग आप पायथन से ग्रोव हार्डवेयर के साथ काम करने के लिए करेंगे।
```
[the remark this answers is not in the document]
Comment (Anurag-0-1-A) [reply 1-getting-started/lessons/1-introduction-to-iot/translations/pi.hi.md:168]: Please Check this again !!

**Review by Anurag-0-1-A — CHANGES_REQUESTED** [review]
Nice Work !!

Just see some issues I've mentioned previously, Please.

**Comment by AdityaGarg00** [comment]
It's okay if that's repeated user will skip that portion it will just help him to explain better and I also discussed it with jim in the original pi.md file.

**Review by Anurag-0-1-A — APPROVED** [review]

**Review by jimbobbennett — APPROVED** [review]

**Comment by jimbobbennett** [comment]
@AdityaGarg00 - can you update your branch from main please? There should be an option in GitHub to do this, but the button is missing for some reason. Once updated I'll merge this.

**Review by jimbobbennett — APPROVED** [review]

**Pull request merged**